client/src/common/components/GridList.jsx

```diff
@@ -70,48 +70,42 @@ class GridList extends PureComponent {
           {name}
         </h2>
         <div className="grid-list__body">
-          {pending ? (
-            <Preloader />
-          ) : (
-            <Fragment>
-              <ul className="grid-list__list">
-                {onAddNew && (
-                  <li className="grid-list__item">
-                    <button
-                      className="grid-list__button"
-                      onClick={onAddNew}
-                      type="button"
-                    >
-                      <CardPlus />
-                    </button>
-                  </li>
-                )}
-                {_map(items, item => (
-                  <li className="grid-list__item" key={item._id}>
-                    <CardItem
-                      color={color}
-                      description={item.description}
-                      doneItemsCount={item.doneItemsCount}
-                      isFavourite={item.isFavourite}
-                      isPrivate={item.isPrivate}
-                      membersCount={item.membersCount}
-                      name={item.name}
-                      onCardClick={this.handleCardClick(route, item._id)}
-                      onFavClick={this.handleFavClick(
-                        item._id,
-                        item.isFavourite
-                      )}
-                      route={route}
-                      unhandledItemsCount={item.unhandledItemsCount}
-                    />
-                  </li>
-                ))}
-              </ul>
-              {_isEmpty(items) && (
-                <MessageBox message={placeholder} type={MessageType.INFO} />
+          <Fragment>
+            <ul className="grid-list__list">
+              {onAddNew && (
+                <li className="grid-list__item">
+                  <button
+                    className="grid-list__button"
+                    onClick={onAddNew}
+                    type="button"
+                  >
+                    <CardPlus />
+                  </button>
+                </li>
               )}
-            </Fragment>
-          )}
+              {_map(items, item => (
+                <li className="grid-list__item" key={item._id}>
+                  <CardItem
+                    color={color}
+                    description={item.description}
+                    doneItemsCount={item.doneItemsCount}
+                    isFavourite={item.isFavourite}
+                    isPrivate={item.isPrivate}
+                    membersCount={item.membersCount}
+                    name={item.name}
```

Comment:
wouldn't be better to pass all item as data prop to CardItem component and destructure it inside?
Would be much cleaner.